test/extensions/geoip_providers/maxmind/config_test.cc

```diff
@@ -392,7 +392,7 @@ TEST_F(MaxmindProviderConfigTest, DifferentProviderInstancesForDifferentProtoCon
   )EOF";
   const auto provider_config_yaml2 = R"EOF(
     common_provider_config:
-      geo_headers_to_add:
+      geo_field_keys:
```

Comment:
again need to make sure that old config/codepath is tested as well, cannot replace 100%.